.github/workflows/feed.yml

```diff
@@ -82,3 +82,39 @@ jobs:
         type: 'stream'
         topic: 'New package registrations'
         content: ${{ steps.zulip.outputs.content }}
+        
+  update-twiter:
+      if: ${{ github.event.label.name == 'new package' && github.repository == github.event.pull_request.head.repo.full_name }}
+      runs-on: ubuntu-latest
+      steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.6.2
```

Comment:
`1.6.2` is a bit odd, it isn't even the latest LTS (that'd be `1.6.7`).  But presumable `1.7.2` (latest stable version` would also work?

Reply:
Works for me!